webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -34,13 +34,17 @@ export type Props = {
     enforceFocus?: boolean;
```

Comment:
Enhancement to the GenericModal component to support the header and footer variations specified in UX Patterns design.